charts/lagoon-logging/Chart.yaml

```diff
@@ -31,6 +31,16 @@ dependencies:
 # It should be started afresh for each release
 # Valid supported kinds are added, changed, deprecated, removed, fixed and security
 annotations:
+  artifacthub.io/containsSecurityUpdates: "true"
   artifacthub.io/changes: |
-    - kind: added
-      description: support for annotations on the CDN service
+    - kind: security
+      description: >
+        Previously the test role and rolebinding were being installed
+        unconditionally. With this change they are only installed when running
+        helm test and then removed.
+
+        Having the role and rolebinding installed unconditionally would mean
+        that any serviceaccount (including the default) would be able to view
+        service objects in the lagoon-logging namespace. This is an unnecessary
+        elevation of privilege, but these service objects do not contain
+        anything sensitive.
```

Comment:
i'd prefer a one-liner description here to keep the artifacthub changelog clearer. IMO this content needs to be in the PR body.

Reply:
Okay, reduced to one line.